packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/ExpoComposeAndroidView.kt

```diff
@@ -0,0 +1,23 @@
+package expo.modules.kotlin.views
+
+import android.view.View
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.viewinterop.AndroidView
+import com.facebook.react.uimanager.PixelUtil.pxToDp
+import expo.modules.kotlin.AppContext
+
+/**
+ * An ExpoComposeView for [AndroidView] wrapping with existing view
+ */
+class ExpoComposeAndroidView(private val view: View, appContext: AppContext) : ExpoComposeView<ComposeProps>(view.context, appContext) {
+  @Composable
+  override fun Content() {
+    AndroidView(
+      factory = { view },
```

Comment:
Are you sure prop updates are passed to the view correctly? In my testing it was impossible to update styling of views mounted inside compose in a similar fashion

Reply:
tested on the code at https://github.com/expo/expo/blob/%40kudo/ui/pure-composeview-3/apps/native-component-list/src/screens/UI/JetpackComposePrimitivesScreen.android.tsx#L28-L31.
it works. change backgroundColor and fast refresh would also update the styles.

if there're other problems i will follow up later